seednode/docker/prod/Dockerfile-btc

```diff
@@ -0,0 +1,7 @@
+FROM ruimarinho/bitcoin-core:0.18
```

Comment:
I think we should prefer building the bitcoin node in the bisq source tree. It's pretty easy to make a Dockerfile for bitcoind, so this dependency is unnecessary